openedx/frontend-app-course-authoring
# style: UX Refinements on taxonomy pages [FC-0036]

This PR fixes/updates the following:

- [X] 1. On the taxonomies list, display the full name of taxonomies in a tooltip if it's longer than what's displayed
![image](https://github.com/openedx/frontend-app-course-authoring/assets/11827305/5200073b-3108-4738-a2e0-a7fa679ea798)
- [X] 2. On the taxonomy detail page, please change the title of the "Value" column to "Tag name"
- [x] 3. On taxonomy detail page, remove the "child tags" column and put it in parentheses instead
- [X] [Update tags count color ](https://github.com/openedx/modular-learning/issues/157#issuecomment-1840330796)
![image](https://github.com/openedx/frontend-app-course-authoring/assets/11827305/845b915e-1323-4dc6-a58e-3a52e242cb3b)
- [x] 4. several minor issues. [See link](https://github.com/openedx/modular-learning/issues/105#issuecomment-1829412705).
![image](https://github.com/openedx/frontend-app-course-authoring/assets/11827305/114730b2-e80b-46d7-85dd-43fafd5ba25c)
![image](https://github.com/openedx/frontend-app-course-authoring/assets/11827305/9174aa69-cb98-43d2-b978-92f9b4b45795)
- [x] 5. Slack: Make your browser window short (not too tall). From the Taxonomy List page, scroll down and click on "WGU" taxonomy. The taxonomy detail page loads, but it is already scrolled down. It should load with scroll=0. Figure out why react-router isn't properly resetting the scroll to 0 when clicking a link, as it should do for every page transition.


You can see the respective section on: https://github.com/openedx/modular-learning/issues/157

------------------------------------

**Testing instructions**

- Enable [taxonomy flags](https://github.com/openedx/frontend-app-course-authoring#feature-taggingtaxonomy-pages).
- Before to all you need to run the [taxonomy-sample-data](https://github.com/open-craft/taxonomy-sample-data).

**1. On the taxonomies list, display the full name of taxonomies in a tooltip if it's longer than what's displayed**

- Go to the [taxonomy list page](http://localhost:2001/taxonomies/) and check that the tooltip is displayed when you hover the title of truncated taxonomies.

**2. On the taxonomy detail page, please change the title of the "Value" column to "Tag name"**

- Go to the taxonomy details page of a taxonomy (click on a taxonomy on the taxonomy list page), and verify the change.

**3. On taxonomy detail page, remove the "child tags" column and put it in parentheses instead**

- Go to the taxonomy details page of a taxonomy and verify the change.

**4. several minor issues.**

- Verify that changes to the link have been made.

**5. Slack [...]**

- Follow the instructions of the issue and verify that the bug is gone.

------------------------------------

**Aditional info**

- Github issue: https://github.com/openedx/modular-learning/issues/157
- Internal ticket: [FAL-3537](https://tasks.opencraft.com/browse/FAL-3537)
- A bug has been introduced in a [PR that is still open](https://github.com/openedx/modular-learning/issues/157#issuecomment-1841021959)
